datamol/match.py

```diff
@@ -0,0 +1,139 @@
+from rdkit.Chem import AllChem
+from typing import Dict, List, Union
+import networkx as nx
+from networkx.algorithms.isomorphism.vf2userfunc import GraphMatcher
+from loguru import logger
+
+
+def mol_to_nx_graph(mol: AllChem.rdchem.Mol) -> nx.Graph:
```

Comment:
Datamol has https://github.com/datamol-org/datamol/blob/main/datamol/graph.py, can you uniformize that and `mol_to_nx_graph`

Also, please make sure this is covered by unit test